src/tests/yaml_configs/test_aws_amsua_2.yaml

```diff
@@ -0,0 +1,11 @@
+s3_bucket: noaa-reanalyses-pds
+s3_prefix: observations/reanalysis/gps/gpsro/%Y/%m/bufr/
```

Comment:
the file name says amsua but it's pointing to gps data? need to make sure the file name of the file and the bufr files is consistent with the s3_prefix